src/amo/components/AddonReviewList/index.js

```diff
@@ -320,25 +329,33 @@ export class AddonReviewListBase extends React.Component<InternalProps> {
           <RatingsByStar addon={addon} />
         </Card>
 
-        <CardList
-          className="AddonReviewList-reviews"
-          footer={paginator}
-          header={reviewCountHTML}
-        >
-          <ul>
-            {allReviews.map((review, index) => {
-              return (
-                <li key={String(index)}>
-                  <AddonReviewCard
-                    addon={addon}
-                    location={location}
-                    review={review}
-                  />
-                </li>
-              );
-            })}
-          </ul>
-        </CardList>
+        <div className="AddonReviewList-reviews">
+          {reviewId && (
+            <FeaturedAddonReview addon={addon} reviewId={reviewId} />
+          )}
+          <CardList
+            className="AddonReviewList-reviews-listing"
+            footer={paginator}
+            header={reviewCountHTML}
+          >
+            <ul>
+              {allReviews.map((review, index) => {
+                if (!reviewId || (review && review.id !== reviewId)) {
```

Comment:
Oh, bummer. This is going to mess up the paginator. Rather than trying to add a bunch of complex code to make the paginator work, I suggest you leave it like this (or maybe don't exclude reviews?) and file an API bug to add a new parameter like `exclude_review_id=N`.

Reply:
I'm not sure what you mean by mess up the paginator. Do you mean that if you click to go to another page you'll no longer see your featured review at the top? If so, then yes, that's how it works now. I wasn't sure how big an issue that was, because if you're going to start paging through reviews then do you really need to still see the featured review at the top?

Reply:
The problem with not excluding the review is that if you look at your own review, and go to edit or delete it, you see the UI in both the FeaturedReviewCard and the first AddonReviewCard in the listing, which looks pretty weird.

Reply:
I opened https://github.com/mozilla/addons-server/issues/9424 to allow us to exclude a review from the API results.